packages/core/src/entity/Reference.ts

```diff
@@ -222,6 +222,32 @@ export class ScalarReference<Value> {
     return this.value;
   }
 
+   /**
+   * Ensures the underlying entity is loaded first (without reloading it if it already is loaded).
+   * Returns the entity or throws an error just like `em.findOneOrFail()` (and respects the same config options).
+   */
```

Comment:
this is wrongly indented

```suggestion
  /**
   * Ensures the underlying entity is loaded first (without reloading it if it already is loaded).
   * Returns the entity or throws an error just like `em.findOneOrFail()` (and respects the same config options).
   */
```